src/tuf/aklocalreposource.cc

```diff
@@ -0,0 +1,30 @@
+#include "aklocalreposource.h"
+
+namespace aklite {
+namespace tuf {
+
+AkLocalRepoSource::AkLocalRepoSource(std::string name_in, std::string local_path) {
```

Comment:
It looks like the given local repo source is not "aklite" specific. If so then `ak` prefix can be removed from its naming.

Reply:
Done.